### ISSUE TITLE
Trimmomatic could not remove adapters

### ISSUE THREAD

**Issue by ddl88** [FilAlves/micro-primers#6]
Hi

I am trying to run the script on my own dataset in command line but i get the following error message: "Error: Trimmomatic could not remove adapters". I have checked that the software is correctly installated by typing "trimmomatic" and it works. I have also tried to run the analysis using the bats file provided as example but I get the same error message. Probably something wrong is happening when specifying the options. I have tried to solve it by myself checking solutions on internet but now I am stuck.
Do you have any suggestion to overcome the issue? Could you please also give me an example of parameter specifications to run the example file on the terminal in order to discard possible issue there?

Thanks a lot!
Best wishes,

Daniele

**Comment by FilAlves** [comment]
Hi Daniele! I just updated micro-primers with regards to trimmomatic, but it shouldn't be the cause of the problem. Even so, i recomend you to update your local installation.

Are you using the micro-primers's conda environment?
Are you using micro-primers in Linux or MacOs?

Additionally, it should be a hidden folder in your micro-primers folder called ".logs". Inside that folder there is the trimmomatic log file, called "trim_log.txt". Can you send the contents of this file?

**Comment by ddl88** [comment]
Hi Felipe! Many thanks for your quick reply. I checked the log file and it turned out that the problem was that the file extension ".fastq" was missing in the command specifying the input and, obviously, the program was not able to find the input file... such elementary...! Once fixed this, I was able to complete the pipeline with the example file and now it is running with my own dataset. (Anyway, I was running the analysis on the Ubuntu app of Windows 10).
Thank you very much for your support and for developing this amazing software! I look forward to citing it in my next work ;)

Best wishes,

Daniele
